test/utils.sh

```diff
@@ -1199,3 +1199,40 @@ retry() {
       sleep "${interval}"
   done
 }
+
+# This function will be used by tasks in tekton-integration-catalog
+# If the image is an index (multi-arch), replace the digest with the 0th manifest's digest; if it's a single-arch image manifest, leave it unchanged.
+resolve_arch_specific_digest() {
+  local FBC_FRAGMENT_IMAGE="$1"
+
+  # Validate input parameters
+  if [[ -z "$FBC_FRAGMENT_IMAGE" ]]; then
+    echo "resolve_arch_specific_digest: Invalid input. Usage: resolve_arch_specific_digest <FBC_FRAGMENT_IMAGE>" >&2
+    exit 2
+  fi
+
+  # Inspect the raw image data
+  local raw_output
+  if ! raw_output=$(retry skopeo inspect --raw "docker://${FBC_FRAGMENT_IMAGE}"); then
+    echo "resolve_arch_specific_digest: Failed to inspect the image: ${FBC_FRAGMENT_IMAGE}" >&2
+    exit 1
+  fi
+
+  # Get manifest count (number of platform-specific variants)
+  local manifest_count
+  manifest_count=$(echo "$raw_output" | jq '.manifests | length')
+
+  if [[ "$manifest_count" -gt 1 ]]; then
```

Comment:
Can we not just use `get_image_manifests`? This returns a map of `{arch:digest}` so we would just always be able to take the first digest in that map after making the call.